1_inventory/src/summarize_wqp_records.R

```diff
@@ -105,3 +105,38 @@ summarize_wqp_data <- function(wqp_inventory_summary_csv, wqp_data, fileout){
   
 }
 
+
+#' @title Summarize WQP records
+#' 
+#' @description 
+#' Function to group WQP data records using any combination of columns, 
+#' summarize the number of records in each group, and save the results 
+#' to a .csv file. 
```

Comment:
I think this function should live in a `3_harmonize/src/` R script since it is called in that phase.

Reply:
Sounds good! Following this logic, I also moved `summarize_wqp_download()` from `1_inventory/src/summarize_wqp_records.R` to a new file in `2_download/src` and edited some of the function/file names.